firestore/integration_test.go

```diff
@@ -151,25 +154,42 @@ func createIndexes(ctx context.Context, dbPath string) {
 				Fields:     adminPbIndexFields,
 			},
 		}
-		go func(req *adminpb.CreateIndexRequest) {
-			op, err := iAdminClient.CreateIndex(ctx, req)
-			if err != nil {
-				log.Fatalf("CreateIndex: %v", err)
+		go func(req *adminpb.CreateIndexRequest, i int) {
+			op, createErr := iAdminClient.CreateIndex(ctx, req)
+			if createErr != nil {
+				log.Fatalf("CreateIndex: %v", createErr)
 			}
 
-			_, err = op.Wait(ctx)
-			if err != nil {
-				log.Fatalf("Wait: %v", err)
+			createdIndex, waitErr := op.Wait(ctx)
+			if waitErr != nil {
+				log.Fatalf("Wait: %v", waitErr)
+			} else {
```

Comment:
log.Fatalf will stop the process, so is this `else` necessary?

Reply:
Removed else